dlhub_sdk/models/__init__.py

```diff
@@ -204,15 +204,24 @@ def set_domains(self, domains):
         self._output["dlhub"]["domains"] = domains
         return self
 
-    def set_visibility(self, visible_to):
-        """Define the list of people and groups who have permissions to see and use this model.
+    def set_visibility(self, users=[], groups=[]):
```

Comment:
Mutable default arguments [can cause some weird errors](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments). Could you change these out for a tuple?